fuzzers/run_fuzzer.py

```diff
@@ -0,0 +1,528 @@
+        int(raw_usage.ru_stime),
+    )
+
+
+def get_load():
+    """Return current load average.
+
+    Values is average perecntage of CPU used over 1 minute, 5 minutes, 15
+    minutes.
+    """
+    a1, a5, a15 = os.getloadavg()
+    cpus = os.cpu_count()
+    return a1 / cpus * 100.0, a5 / cpus * 100.0, a15 / cpus * 100.0
+
+
+_pstree_features = None
+
+
+def get_pstree_features():
+    global _pstree_features
+    if _pstree_features is None:
+        _pstree_features = []
+        for f in ['-T', '-l']:
+            try:
+                subprocess.check_output(
+                    "pstree {}".format(f),
+                    stderr=subprocess.STDOUT,
+                    shell=True,
+                )
+                _pstree_features.append(f)
+            except subprocess.CalledProcessError:
+                continue
+    return _pstree_features
+
+
+def get_pstree(pid):
+    """Get processes under current one.
+
+    Requires the pstree until be installed, otherwise returns empty string.
+    """
+    p = subprocess.Popen(
+        "pstree {} {}".format(" ".join(get_pstree_features()), pid),
+        shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        stdin=subprocess.DEVNULL,
+        universal_newlines=True,
+    )
+    stdout, stderr = p.communicate()
+
+    # Special case segfaults.
+    if 'Segmentation fault' in stderr:
+        stderr = ''
+        stdout = '(result to big, pstree segfaulted.)'
+
+    # If no error, just return stdout
+    if not stderr:
+        return stdout
+
+    return "{}\n{}".format(stdout, stderr)
+
+
+def run_submake(make_flags):
+    """Check if make_flags indicate that we should execute things.
+
+    See https://www.gnu.org/software/make/manual/html_node/Instead-of-Execution.html#Instead-of-Execution  # noqa
+
+    If this is a dry run or question then we shouldn't execute or output
+    anything.
+
+    The flags end up as single letter versions in the MAKEFLAGS environment
+    variable.
+
+    >>> run_submake('')
+    True
+
+    The following flags are important;
+
+     -n == --dry-run
+
+    >>> run_submake('n')
+    False
+    >>> run_submake('n --blah')
+    False
+    >>> run_submake('--blah n')
+    False
+    >>> run_submake('--blah')
+    True
+    >>> run_submake('--random')
+    True
+
+     -q == --question
+
+    >>> run_submake('q')
+    False
+    >>> run_submake('q --blah')
+    False
+    >>> run_submake('--blah q')
+    False
+    >>> run_submake('--blah')
+    True
+    >>> run_submake('--random')
+    True
+
+      Both --dry-run and --question
+
+    >>> run_submake('qn')
+    False
+    >>> run_submake('nq')
+    False
+    >>> run_submake('--quiant')
+    True
+    """
+    r = re.search(r'(?:^|\s)[^-]*(n|q)[^\s]*(\s|$)', make_flags)
+    if not r:
+        return True
+    return not bool(r.groups()[0])
+
+
+def main(argv):
+    fuzzers = os.path.abspath(os.path.dirname(__file__))
+
+    assert len(argv) == 2, argv
```

Comment:
Use argparse and add documentation for tool and it's arguments

Reply:
Done.